tools/system_libs.py

```diff
@@ -1581,6 +1581,8 @@ class libcxx(NoExceptLibrary, MTLibrary):
     '-DLIBCXX_BUILDING_LIBCXXABI=1',
     '-D_LIBCPP_BUILDING_LIBRARY',
     '-D_LIBCPP_DISABLE_VISIBILITY_ANNOTATIONS',
+    # This increases too much code size
+    '-D_LIBCPP_AVAILABILITY_HAS_NO_VERBOSE_ABORT',
```

Comment:
Can this be removed now that its in the config_site?

Reply:
Ah right, forgot to do that.